### DISCUSSION TITLE
is it possible to make a password for my robots?

### DISCUSSION THREAD

**Discussion by Potato-eater** [pybricks/support#1120] [Q&A] [answered]
I have a robots installed with pybricks, but I have noticed that anyone can connect to my robot (which I do not want). A while ago, I was in a robotics competition. Someone else connected to my robot, and it ran off the table that it was on. It ended up being shattered into pieces. 
I hope that maybe the robot would require a password to connect to, instead of how anyone can connect to any robot.

**Comment by dlech** [comment, marked as answer]
Pybricks doesn't currently support this but it is something we could consider in the future.

**Comment by Kermit647** [comment]
With the possibility to disable bluetooth, this would perhaps not be neccesary anymore.
